### PR TITLE
WDS-72 - Badge component

### PR DESCRIPTION
**Pull request by timofeytrepalin**

### CODE REVIEW

File: packages/theme/fonts/Sora/index.css
Comment (0x009922) [inline packages/theme/fonts/Sora/index.css:7]: Why 700?

File: packages/ui/src/components/Badge/Badge.vue
Comment (0x009922) [inline packages/ui/src/components/Badge/Badge.vue:33]: I am a bit confused about `s-border` class. It is specific for Badge component, so, according to BEM, it should be named as `s-badge--border` or `s-badge--bordered` or something. But in this case it is OK because of `scoped` style. But if it is scoped, why to use BEM at all?
Also I am not sure about usability of `scoped` in terms of component library, especially when it is about such a small component as Badge one which is the best pretender to be refactored as functional component (for performance reasons) in future. `scoped` style doesn't work with functional components. So, it is my point to not use `scoped` with components that are almost stateless.
Anyway, I am just confused. That's not a critical note, but if I were you, I would like to drop `scoped` and transform `s-border` -> `s-badge--border`.

File: packages/theme/fonts/Sora/index.css
Comment (0x009922) [reply packages/theme/fonts/Sora/index.css:7]: Btw, I am planning to fix this font as part as WDS-70

**Review by 0x009922 — APPROVED** [review]

**Review by 0x009922 — CHANGES_REQUESTED** [review]
Please fix formatting

File: packages/ui/src/components/Badge/Badge.vue
Comment (stefashkaa) [inline packages/ui/src/components/Badge/Badge.vue:4]: `BdgType` -> `BadgeType` sounds better

File: packages/ui/src/components/Badge/Badge.vue
Comment (stefashkaa) [inline packages/ui/src/components/Badge/Badge.vue:29]: do we need an empty class here?

File: packages/ui/stories/Badge.stories.ts
Comment (stefashkaa) [inline packages/ui/stories/Badge.stories.ts:12]: `BadgeType` can be used here

File: packages/ui/stories/Badge.stories.ts
Comment (stefashkaa) [inline packages/ui/stories/Badge.stories.ts:40]: it can be an array of values from `BadgeType`

**Review by stefashkaa — APPROVED** [review]

File: packages/ui/stories/Badge.stories.ts
Comment (0x009922) [reply packages/ui/stories/Badge.stories.ts:12]: +

**Review by 0x009922 — CHANGES_REQUESTED** [review]
`next` branch now contains new design system implementation. Update this PR due to this, please.

File: .changeset/pretty-gorillas-develop.md
Comment (0x009922) [inline .changeset/pretty-gorillas-develop.md:3]: This change isn't related to the `theme` package.

File: packages/theme/src/sass/tokens.scss
Comment (0x009922) [inline packages/theme/src/sass/tokens.scss:33]: We don't have such token in the Figma. Where does it come from?

File: packages/theme/fonts/Sora/index.css
Comment (0x009922) [inline packages/theme/fonts/Sora/index.css:7]: Could you explain why did you add it?
For me it looks unnecessary and breaking - Sora font is already defined at `theme/src/fonts/Sora` and could be imported as described at README. Woff2 & woff break variations, so only ttf is fine.

File: packages/ui/src/components/Badge/Badge.vue
Comment (0x009922) [inline packages/ui/src/components/Badge/Badge.vue:37]: We don't have `text-branch-pms-black` utility - now our Windi config contain only default utilities.

File: packages/ui/src/components/Badge/Badge.vue
Comment (0x009922) [inline packages/ui/src/components/Badge/Badge.vue:71]: Looks like you forget to use typography token correctly. Please see README at the theme package.

File: .changeset/config.json
Comment (0x009922) [inline .changeset/config.json:7]: Please remove this - I'll create a separated PR with a fix for it.

File: Jenkinsfile
Comment (0x009922) [inline Jenkinsfile:7]: Remove this change too, please. I'll fix it in a separated PR.

**Review by 0x009922 — CHANGES_REQUESTED** [review]

File: packages/theme/fonts/Sora/index.css
Comment (timofeytrepalin) [reply packages/theme/fonts/Sora/index.css:7]: I guess it is remnant after auto merging, I'll fix it

File: packages/theme/src/sass/tokens.scss
Comment (timofeytrepalin) [reply packages/theme/src/sass/tokens.scss:33]: It is color from figma, for DEBUG status
Should I add this color locally, in component Badge.vue, instead?

Comment (0x009922) [reply packages/theme/src/sass/tokens.scss:33]: Hmm, I see. I think it is a bug of the design in Figma. I've reported it.

**Review by 0x009922 — COMMENTED** [review]
I can't approve because badge DEBUG color is not resolved yet. (waiting for the response from the designer)

File: packages/ui/src/components/Badge/Badge.vue
Comment (0x009922) [inline packages/ui/src/components/Badge/Badge.vue:45]: Bad BEM - `s-badge--border` -> `s-badge_border`, `s-badge--color-${type}` -> `s-badge_color_${type}`

File: packages/ui/src/components/Badge/types.ts
Comment (0x009922) [inline packages/ui/src/components/Badge/types.ts:10]: Unnecessary duplication and enum `BadgeTypes` is useless because its values are numbers and not actual strings.

Comment (0x009922) [reply packages/ui/src/components/Badge/types.ts:10]: https://www.typescriptlang.org/play?#code/KYOwrgtgBAQghgEwObACoE8AOwoG8CwAUFCVHAMYAuAlgG7AA0RpU1IFN9RAvkA

File: packages/ui/src/components/Badge/Badge.vue
Comment (0x009922) [inline packages/ui/src/components/Badge/Badge.vue:33]: Why is it tabbable?

**Review by 0x009922 — COMMENTED** [review]
Also, I don't see a typography configuration

**Review by 0x009922 — APPROVED** [review]

**Pull request merged**
